api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 10.0
 
+**Feature level 284**
+
+* [`POST /users/{user_id}/subscription`](/api/subscribe): Returns two pairs of
+  dictionaries where the key is the id of the user/bot and the value is a list
+  of the names of the channels that were subscribed/already subscribed to as a
+  result of the query.
```

Comment:
Here in the changelog, the description should focus on the contrast from the old behavior to the new behavior. Effectively it's a form of diff.

So e.g.,
```suggestion
* [`POST /users/{user_id}/subscription`](/api/subscribe): In the response,
  users are identified by their numeric user ID rather than by their
  Zulip API email address.
```

Reply:
Applied in latest commit.